projects/hslayers/src/components/map/map.service.ts

```diff
@@ -1198,19 +1198,19 @@ export class HsMapService {
 
   /**
    * Remove all layers gained from composition from map
+   * @param force Remove all removable layers no matter fromComposition param
    * @param app - App identifier
    */
-  removeCompositionLayers(app: string): void {
-    const to_be_removed = [];
-    this.getLayersArray(app).forEach((lyr) => {
-      if (
-        getFromComposition(lyr) ||
-        getRemovable(lyr) === undefined ||
-        getRemovable(lyr) == true
-      ) {
-        to_be_removed.push(lyr);
-      }
-    });
+  removeCompositionLayers(force: boolean, app: string): void {
```

Comment:
Could you consider changing the method's signature to
```suggestion
  removeCompositionLayers({force, app}: {force: boolean; app: string}): void {
```
also perhaps while setting some sane defaults, effectively making the params optional.

It is so much more readable to call it like `removeCompositionLayers({force: true})`
instead of `removeCompositionLayers(true, 'app')`

Reply:
Good point, will refactor.

Reply:
Take a look here 0bf8d75babe1ef01270508bc50d43ac93ffab1cc. 

Made force param optional effectively setting its default to false.
Also tried documenting it properly. More or less it works. Param description is infered from interface
![image](https://user-images.githubusercontent.com/44566616/206199363-1e55422b-3d03-40f4-b99b-58c431cb0a41.png)

Reply:
Wow, so much profi with the interface! Thanks!